File: CRS-Signify-SpringRest-Group-G/src/main/java/com/signify/bean/Student.java
package com.signify.bean;

import java.io.Serializable;

public class Student extends User implements Serializable {
	private int age;
	private int isApproved;
	private String studentId, fname, bloodgroup, phnum, name, branch;

	public int getIsApproved() {
		return isApproved;
	}

	public void setIsApproved(int isApproved) {
		this.isApproved = isApproved;
	}

	public String getBranch() {
		return branch;
	}

	public void setBranch(String branch) {
		this.branch = branch;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}

	public String getStudentId() {
		return studentId;
	}

	public void setStudentId(String studentId) {
		this.studentId = studentId;
	}

	public String getFname() {
		return fname;
	}

	public void setFname(String fname) {
		this.fname = fname;
	}

	public String getBloodgroup() {
		return bloodgroup;
	}

	public void setBloodgroup(String bloodgroup) {
		this.bloodgroup = bloodgroup;
	}

	public String getPhnum() {
		return phnum;
	}

	public void setPhnum(String phnum) {
		this.phnum = phnum;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int isApproved() {
		return isApproved;
	}

	public void setApproved(int isApproved) {
		this.isApproved = isApproved;
	}

}

File: CRS-Signify-UI-UX-Group-G/src/app/model/student.ts
export class Student {
    public studentId: string;
    public age: number;
    public isApproved: number;
    public studentid: string;
    public fname: string;
    public bloodgroup: string;
    public phnum: string;
    public name: string;
    public branch: string;

    constructor(
        studentId: string,
        age: number,
        isApproved: number,
        studentid: string,
        fname: string,
        bloodgroup: string,
        phnum: string,
        name: string,
        branch: string,
        userid: string,
        username: string,
        address: string,
        password: string,
        doj: string,
        roleid: number
    ) {
        this.studentId = studentId;
        this.age = age;
        this.isApproved = isApproved;
        this.studentid = studentid;
        this.fname = fname;
        this.bloodgroup = bloodgroup;
        this.phnum = phnum;
        this.name = name;
        this.branch = branch;
    }
}